feat(curriculum): add Artifact Provenance Auditor workshop (#68860)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-artifact-provenance-auditor/6a57c331ae94fa4631b60861.md

---
id: 6a57c331ae94fa4631b60861
title: Step 2
challengeType: 1
dashedName: step-2
---

# --description--

> Artifacts have a person responsible for them. The value of a property can be another object.

Reviewer: ```suggestion
Each artifact has a person responsible for it. The value of a property can be another object.
```

Add a `curator` property to the artifact stored under `101`. Set its value to an object with an `id` of `201` and a `name` of `"Earl Sinclair"`.

# --hints--

The artifact stored under `101` should have a `curator` property that is an object.

```js
assert.isObject(collection[101].curator);
```

The `curator` should have an `id` of `201`.

```js
assert.strictEqual(collection[101].curator.id, 201);
```

The `curator` should have a `name` of `"Earl Sinclair"`.

```js
assert.strictEqual(collection[101].curator.name, "Earl Sinclair");
```

# --seed--

## --seed-contents--

```js
const collection = {
  101: {
    title: "Golden Mask",
    category: "Ceremonial",
--fcc-editable-region--

> 

Reviewer: Indent the empty line b/w the region markers with 4 spaces.

---
id: 6a57c331ae94fa4631b60861
title: Step 2
challengeType: 1
dashedName: step-2
---

# --description--

> Each artifact has a person responsible for them. The value of a property can be another object.

Reviewer: ```suggestion
Every artifact has a person responsible for it. Remember that the value of a property can also be another object.
```

> Add a `curator` property to the artifact stored under `101`. Set its value to an object with an `id` of `201` and a `name` of `"Earl Sinclair"`.

Reviewer: ```suggestion
Add a `curator` property to the artifact stored under the key `101`, and set its value to an object with an `id` of `201` and a `name` of `"Earl Sinclair"`.
```
